install.sh

```diff
@@ -5,17 +5,8 @@ set -e
 # Install submodules.
 ./install-repositories.sh
 
-# Install KEEP-CORE.
-./install-keep-core.sh
-
-# Install KEEP-ECDSA.
-./install-keep-ecdsa.sh
-
-# Install tBTC.
-./install-tbtc.sh
-
-# Install tBTC dApp.
-./install-tbtc-dapp.sh
+# Install applications and their contracts
+./install-applications-with-contracts.sh
 
 # Do not install keep dashboard dApp for e2e nightly test
 if [[ $E2E_TEST != true ]]
```

Comment:
What do you think about moving this 'if' statement along with `./install-keep-dashboard.sh` to `install-applications-with-contracts.sh` as well? This way, Keep Dashboard dApp installation script will be in the same place as tBTC dApp. One can pass env var `E2E_TEST=true ./install.sh` when working only in tBTC world and doesn't need Keep Dashboard.

Also, I'm wondering if it makes sense to rename $E2E_TEST -> $INCLUDE_KEEP_DASHBOARD, ie:
```
# Install keep dashboard dApp when it is needed. In most cases we do not need
# Keep Dashboard when working on e2e tests or working on tBTC.
if [[ $INCLUDE_KEEP_DASHBOARD == true ]]
then
    # Install Keep Dashboard.
    ./install-keep-dashboard.sh
fi
```

Reply:
> What do you think about moving this 'if' statement along with ./install-keep-dashboard.sh to install-applications-with-contracts.sh as well?

:100:

> I'm wondering if it makes sense to rename $E2E_TEST -> $INCLUDE_KEEP_DASHBOARD

My only hesitation there would be the case where `$E2E_TEST` is being used in our testing scripts or build tool scripts or something

Reply:
Yes, you're right! We have one reference in [here](https://github.com/keep-network/local-setup/blob/b22089e029f097b4beb8728a049de42912a9b084/.circleci/config.yml#L72) and I think this is all. I've introduced `E2E_TEST` [here](https://github.com/keep-network/local-setup/commit/f36929ea3d6f4b91d0fa578ab54256195437873c). Most likely that's the only change, cause I just wanted to speed up the `local-setup` setup.

Reply:
Gotcha - In that case I think it would be better if we scoped this PR to just the script changes and leave refactoring `$E2E_TEST` for another PR. That okay with you?

Reply:
Sure, changes around scripts look good to me!

Reply:
Created issue to rename `$E2E_TEST` https://github.com/keep-network/local-setup/issues/63